### PR TITLE
feat: add demo request entry

### PR DESCRIPTION
**Pull request by Megrax**
| Before | After |
| :---: | :---: |
| CleanShot 2022-08-09 at 15 09 51 | CleanShot 2022-08-09 at 15 08 27 |

### Details

| Before Submit | After Submit |
| :---: | :---: |
| CleanShot 2022-08-09 at 15 12 57 | CleanShot 2022-08-09 at 15 12 41 |

### CODE REVIEW

**Review by boojack — APPROVED** [review]
LGTM

File: pages/demo/index.vue [hunk 1]
```diff
@@ -117,16 +142,37 @@ export default defineComponent({
         eventLabel: parts[2],
       });
       // Segment
-      analytics.value?.track(name);
+      analytics.other.track(name);
+    };
+
+    const subscribe = (e: any) => {
+      trackEvent("demo");
+      analytics.newsletter.identify(email.value, {
```
Comment (ecmadao) [inline pages/demo/index.vue:150]: How to distinguish if the user just wants to subscribe to the newsletter?

**Review by ecmadao — APPROVED** [review]
LGTM otherwise

File: locales/zh.json [hunk 2]
```diff
@@ -394,6 +394,8 @@
       }
     },
     "learn-more": "视频演示",
-    "try-it-now": "马上试试"
+    "appoint-demo": "预约演示",
+    "appointment-success": "预约成功",
+    "appointment-success-description": "我们将尽快联系你"
```
Comment (tianzhou) [inline locales/zh.json:399]:
```suggestion
    "appointment-success-description": "我们将在 1 个工作日内联系您！"
```

File: locales/en.json [hunk 3]
```diff
@@ -394,6 +394,8 @@
       }
     },
     "learn-more": "Learn More",
-    "try-it-now": "Try it now"
+    "appoint-demo": "Appoint Demo",
+    "appointment-success": "Congratulations!",
+    "appointment-success-description": "We will contact you soon"
```
Comment (tianzhou) [inline locales/en.json:399]:
```suggestion
    "appointment-success-description": "We will contact you in 1 business day!"
```

File: pages/demo/index.vue (hunk 1 above)
Comment (Megrax) [reply pages/demo/index.vue:150]: We cannot. But we are cheating newsletter subscribers and demo requesters the same way. On MailChimp, there are added to the same audience list.

**Review by tianzhou — APPROVED** [review]
LGTM

**Comment by Megrax** [comment]
PTAL

**Review by tianzhou — APPROVED** [review]
LGTM

File: pages/demo/index.vue (hunk 1 above)
Comment (Megrax) [reply pages/demo/index.vue:150]: If users want to subscribe to our newsletter, we'll also offer them the live demo.

**Pull request merged**